web/src/utils/url.ts

```diff
@@ -12,7 +13,11 @@ export const getTimeOfInterestFromUrl = (search: string): string => {
     return queryString
   }
 
-  return formatDateInISO(new Date())
+  const currInPST = moment(new Date())
```

Comment:
I would point at that moment recommends that projects shouldn't use it anymore:  https://momentjs.com/docs/#/use-it/